### ISSUE TITLE
Make tests more human-readable

### ISSUE THREAD

**Issue by andylolz** [pwyf/foxpath#14]
One barrier to understanding the tests is simply that they’re not expressed in a very human-readable way. A couple of things could help here:

 - [x] putting the conditionals at the start i.e. instead of ‘… (if …)?’, use ‘if … then …’. Seems a matter of preference (and maybe not very pythonic) but I find this clearer
 - [x] phrasing the tests as expectations, rather than questions. E.g. instead of ‘x exists?’, use ‘x should exist’ or ‘we expect x to exist’
 - [x] rewording to use plain english. So e.g. rather than ‘is on list L?’, use ‘should be on the L codelist’
 - [x] allowing arbitrary whitespace, including carriage returns
 - [x] adding backticks around xpath expressions (refs #7)
 - [x] for tests.csv readability specifically, I’d propose using a gherkin-like YAML file. That would also allow for structuring the tests by indicator

**Comment by andylolz** [comment]
Here’s a sample of a more readable tests.yaml:

```yaml
- indicator-description: Aid Type
  indicator-name: aid-type
  tests:
   - description: Default aid type
     level: 1
     expression: >
      if `activity-status/@code` is at least 2
      then `default-aid-type` should be present
      or `transaction/aid-type` should be present

   - description: Aid type uses standard codelist
     level: 1
     expression: >
      if `activity-status/@code` is at least 2
      then every `default-aid-type/@code` should be on the AidType codelist
      and every `transaction/aid-type/@code` should be on the AidType codelist
```

**Comment by andylolz** [comment]
Fixed in #10.
